client/src/components/Fit.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+function Fit({ rating }) {
+  const [score] = useState((rating / 1) * 100);
+
+  return (
+    <Wrapper>
+
+      <Title>Overall Fit Rating</Title>
+      <Slider>
+        <ValueBox inputWidth={score} />
+        <Divider />
+
+      </Slider>
+
+      <SmallTag>Runs Small</SmallTag>
+      <LargeTag>Runs Large</LargeTag>
```

Comment:
if these are tests of the css you may want to remove them (if they are place holders disregard )